pages/reference/OS/configuration.md

```diff
@@ -0,0 +1,193 @@
+---
+title: Configuration
+excerpt: Instructions for editing and customizing config.json for modifying the behavior of {{ $names.os.lower }}
+---
+
+# Configuring {{ $names.os.lower }}
+
+The behavior of {{ $names.os.lower }} can be configured by editing the `config.json` file. This file is located in the [boot partition][boot-partition] and accepts a [range of fields](#valid-fields) to modify the behavior of the host OS.
+
+**Note:** The `config.json` file is different from the [`config.txt`][config-txt] file, also located in the boot partition, which is used by the Raspberry Pi to set device configuration options.
```

Comment:
I wonder if it's worth mentioning "...which is (only) used by the Raspberry Pi to...", to highlight the fact that non-RaspberryPi devices don't use `config.txt`? :man_shrugging:

Reply:
Thanks for the feedback. I decided to leave this one out as I think it is mostly redundant given the context here is more to draw attention to the fact a similar filename exists.